backend/server/controllers/codeReviews.js

```diff
@@ -23,15 +23,15 @@ module.exports = {
       }
       const allStudentInstancesIds = [] // Gather all student instance ids for future query
       const values = req.body.codeReviews.map((codeReview) => {
-        if (typeof codeReview.reviewer !== 'number' || typeof codeReview.toReview !== 'number') {
+        if (typeof codeReview.reviewer !== 'number' || (typeof codeReview.toReview !== 'number' && !codeReview.repoToReview.includes('http'))) {
           return null // This will lead to rejection later.
         }
         allStudentInstancesIds.push(codeReview.reviewer)
-        allStudentInstancesIds.push(codeReview.toReview)
         return {
           studentInstanceId: codeReview.reviewer,
           reviewNumber: req.body.reviewNumber,
-          toReview: codeReview.toReview
+          toReview: Number.isInteger(codeReview.toReview) ? codeReview.toReview : null,
+          repoToReview: Number.isInteger(codeReview) ? null : codeReview.repoToReview
```

Comment:
`Number.isInteger(codeReview)` is never true

Reply:
Sorry for my carelessness, it should be `Number.isInteger(codeReview.toReview)`